pom.xml

```diff
@@ -56,6 +57,19 @@
         <relativePath/>
     </parent>
 
+    <repositories>
+
+        <repository>
```

Comment:
Looks like the addition of the new repository in pom.xml is not necessary as `spring-security-saml2-service-provider` appears to be in maven central. Please remove the newly added repositories tag

Reply:
Spring Security SAML2 relies on OpenSAML libraries which are not hosted on maven central.  We could also use https://build.shibboleth.net/maven/releases/